jobapi/doc.go

```diff
@@ -0,0 +1,3 @@
+// jobapi provides an API with which to interact with and mutate the currently executing job
+// Pronunciation: /ˈdʒɑbe̞pi/ /joh-bah-pee/
```

Comment:
[nit]

the IPA here is wrong: 
```suggestion
// Pronunciation: /ˈdʒɑbapi/ /joh-bah-pee/
```